### PR TITLE
Complete database setup and seed

### PR DESCRIPTION
**Pull request by RamKhalsa**

### CODE REVIEW

File: client/src/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import ReactDOM from 'react-dom';
+
```
Comment (aku2nnya) [inline client/src/index.js]: @RamKhalsa Is this rendering? I believe you'll need to import React when using JSX

File: client/src/index.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+import BookingModule from './components/BookingModule';
```
Comment (aku2nnya) [inline client/src/index.js]: @RamKhalsa and this? the file type isn't defined (.jsx), but is this still rendering correctly>?

File: client/src/index.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+ReactDOM.render(<BookingModule />, document.getElementById('booking-module'));
```
Comment (aku2nnya) [inline client/src/index.js]: @RamKhalsa Please add a new line break to meet style guide

File: database/index.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+const Sequelize = require('sequelize');
+const pw = require('./config/sequelize.config');
+
+
+const db = new Sequelize('bookings', 'root', pw.rootPW, {
+  host: 'localhost',
+  dialect: 'mysql',
+});
+
+module.exports = db;
```
Comment (aku2nnya) [inline database/index.js]: @RamKhalsa Please add a new line break to meet style guide

File: client/src/index.js (hunk 2 above)
Comment (RamKhalsa) [reply client/src/index.js]: Yup--this works just fine because of my webpack config :)

File: client/src/index.js (hunk 1 above)
Comment (RamKhalsa) [reply client/src/index.js]: thanks for the note--I had tested it as airbnb eslint was mad at having React, but I added it back!

**Pull request closed without merging**